### PR TITLE
iTop.iTopDataRecovery version 3.0.0.177

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/45713)

### CODE REVIEW

File: manifests/i/iTopDataRecovery/iTopDataRecovery/3.0.0.177/iTopDataRecovery.iTopDataRecovery.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: iTopDataRecovery.iTopDataRecovery
+PackageVersion: 3.0.0.177
+PackageLocale: en-US
+Publisher: iTop
+PackageName: iTop Data Recovery
+License: © iTop Inc. All rights reserved.
```
Comment (Trenly) [inline manifests/i/iTopDataRecovery/iTopDataRecovery/3.0.0.177/iTopDataRecovery.iTopDataRecovery.locale.en-US.yaml:9]: A copyright isn't a license;
```suggestion
License: Proprietary
LicenseUrl: https://www.itopvpn.com/eula
Copyright: © iTop Inc. All rights reserved. 
```

File: manifests/i/iTopDataRecovery/iTopDataRecovery/3.0.0.177/iTopDataRecovery.iTopDataRecovery.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: iTopDataRecovery.iTopDataRecovery
+PackageVersion: 3.0.0.177
+PackageLocale: en-US
+Publisher: iTop
```
Comment (Trenly) [inline manifests/i/iTopDataRecovery/iTopDataRecovery/3.0.0.177/iTopDataRecovery.iTopDataRecovery.locale.en-US.yaml:7]: This field must match the value in control panel
```suggestion
Publisher: iTop Inc.
```

File: manifests/i/iTopDataRecovery/iTopDataRecovery/3.0.0.177/iTopDataRecovery.iTopDataRecovery.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: iTopDataRecovery.iTopDataRecovery
+PackageVersion: 3.0.0.177
+Installers:
+- Architecture: neutral
+  InstallerType: inno
+  InstallerUrl: https://goto.itopvpn.com/downloadcenter?product=idr
```
Comment (Trenly) [inline manifests/i/iTopDataRecovery/iTopDataRecovery/3.0.0.177/iTopDataRecovery.iTopDataRecovery.installer.yaml:9]: Better to use the direct URL than a redirected one
```suggestion
  InstallerUrl: https://download.itopvpn.com/dl/itop-data-recovery-setup.exe
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/45713#discussion_r803739992

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
You will need to update the file names to match the new PackageIdentifier and to fall under `/manifests/i/iTop/iTopDataRecovery/3.0.0.177`

**Comment by BajajAditya123** [comment]
done

**Review by OfficialEsco — APPROVED** [review]

**Comment by BajajAditya123** [comment]
cool

**Pull request merged**